### DISCUSSION TITLE
How not use schema?

### DISCUSSION THREAD

**Discussion by anarkrypto** [next-safe-action/next-safe-action#186] [General]
I completely understand that validate the payload is one of the fundamentals of the projects.

However, I have an application where it's already validated it in the client, so I don't want to repeat the validation

However, I would like to have other benefits like hooks, error handlers.

How can I use and type my params without schema?

```typescript
export const signUpWithPassword = safeAction.action<{ email: string, password: string}>(
	async ({ parsedInput: { email, password } }) => {
		await client.auth.signUpWithEmailAndPassword({
			email,
			password,
		})
	},
)
```

I accept other suggestions. Thanks

**Comment by TheEdoRan** [comment]
Check out this discussion and the comment I left there:

- https://github.com/TheEdoRan/next-safe-action/discussions/192#discussioncomment-9973745
